gui/options.cpp

```diff
@@ -244,7 +245,7 @@ void OptionsDialog::build() {
 			int value =  ConfMan.getInt("kbdmouse_speed", _domain);
 			if (_kbdMouseSpeedSlider && value < sizeof(kbdMouseSpeedLabels)) {
```

Comment:
This shouldn't be `sizeof`, but `ARRAYSIZE(kbdMouseSpeedLabels)`. Or maybe `- 1` to avoid the 0 terminator. You should probably also check if `value >= 0`.